helm-charts/README.md

```diff
@@ -1,17 +1,35 @@
-### helm repository Setup
-Assuming you have helm already installed, it is straight-forward to add a Cloudsmith-based chart repository:
+# Infisical Helm Charts
 
-```
-helm repo add infisical-helm-charts 'https://dl.cloudsmith.io/public/infisical/helm-charts/helm/charts/' 
+Welcome to Infisical Helm Charts repository! Find below the instrcutions to setup and install our charts.
+
+```sh
+# Add the Infisical repository
+helm repo add infisical 'https://dl.cloudsmith.io/public/infisical/helm-charts/helm/charts/' && helm repo update
+
+# Install Infisical
+helm upgrade --install --atomic \
+  -n infisical-dev --create-namespace \
+  infisical infisical/infisical
   
-helm repo update
+# Install Infisical Secrets Operator
+helm upgrade --install --atomic \
+  -n infisical-dev --create-namespace \
+  infisical-secrets-operator infisical/secrets-operator
 ```
 
-### Installing a Helm Chart
-```
-helm install infisical-helm-charts/<name-of-helm-chart>
-```
+## Charts
+
+Here's the link to our charts corresponding documentation :
+- **`[infisical](./infisical/README.md)`**
+- **`secrets-operator`**
+
+## Documentation
+
+We're trying to follow a documentation convention across our charts, allowing us to auto-generate markdown documentation thanks to [this tool](https://github.com/bitnami-labs/readme-generator-for-helm)
 
-#### Available chart names
-- infisical
-- secrets-operator
+Steps to update the documentation :
+1. `git clone https://github.com/bitnami-labs/readme-generator-for-helm`
+2. `npm install ./readme-generator-for-helm`
```

Comment:
Can't seem to run this command. It says it can't find the file, which is true since I don't see the file inside of the cloned repo. How did you run it?

Reply:
just tested it, works well